docs/user-guide/ze-usage.md

```diff
@@ -6,23 +6,24 @@ Review this section to familiarize yourself with Zowe Explorer and make the best
 
 ### Operating systems
 
-- Windows 10+
+
 - MacOS 10.15 (Catalina), 11 (Big Sur), 12 (Monterey)
 - Unix-like:
    - [CentOS](https://www.centos.org/) 8+
    - [RHEL](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) 8+
    - [Ubuntu](https://ubuntu.com/) 20.04+
+- Windows 10+
 
-### Integrated development environments:
+### Integrated development environments: 
 
-- [Theia](https://theia-ide.org/) 18+
-- [Red Hat CodeReady Workspaces](https://www.redhat.com/en/technologies/jboss-middleware/codeready-workspaces)
 - [Eclipse Che](https://www.eclipse.org/che/)
+- [Red Hat CodeReady Workspaces](https://www.redhat.com/en/technologies/jboss-middleware/codeready-workspaces)
+- [Theia](https://theia-ide.org/) 18+
 
 ### Platform versions:
```

Comment:
I would suggest removing this "Platform versions" section unless others object:
* Node.js and Yarn do not apply - they are probably left over from a section of developer requirements. For end users of ZE, they shouldn't have to care about either of these.
* VS Code is another development environment (alternative to Theia or Che) and could be moved to the list above. Perhaps as the first item in that list? I know the lists are mostly alphabetized, but VS Code kind of has priority as the primary environment.

Reply:
Updated the content as Timothy advised - thank you!